projects/igniteui-angular/src/lib/core/styles/components/navdrawer/_navdrawer-theme.scss

```diff
@@ -319,13 +319,23 @@
         justify-content: flex-start;
 
         > igx-icon + span {
-            margin-#{$left}: rem(32px);
+            @if $variant == 'indigo-design' {
+                margin-#{$left}: rem(28px);
+            } @else {
+                margin-#{$left}: rem(32px);
+            }
+        }
+
+        span {
+            @if $variant == 'indigo-design' {
+                padding-#{$left}: rem(4px);
```

Comment:
The amount of padding is almost unnoticeable. Did you check with @StefanIvanov?

Reply:
Looks like this is the padding on the left of the icon and if that is the case yes we tested it for a variety of icon shapes to make sure that it does not look broken with any shape and at the same time, it is not too much so that we sacrifice the available width for the content.

Reply:
The padding in question is not the padding when you have icons to the left or right(rtl), it's for when you don't. And it's related to the text, not the icon...

Reply:
I think 8px will be enough when there is no icon.

Reply:
I concur. I tested and 8px looks much better.